src/index.js

```diff
@@ -17,7 +14,7 @@ const display = () => {
     // Creating list of to-do
     const li = document.createElement('li');
     li.innerHTML = `
-    <input type="checkbox">
+    <input class="checkbox" type="checkbox">
```

Comment:
- [ ] You did a great job working on the interactivity of the app. :+1: I believe completed tasks should be checked when reloading the page. Kindly work on it. Currently after reloading the page, the checkboxes reset even if the value in the local storage is `completed: true`. I think the following will get the job done. 🤓️

**Example**
![image](https://user-images.githubusercontent.com/17243325/236408653-6319de54-9f40-44a0-9e45-d853ff0920eb.png)

Reply:
Thanks for the feedback I have fixed the issue so everything is working perfectly fine now.Happy to move to my next project

Reply:
- [ ] Great work 👏🏾